pubber/src/main/java/daq/pubber/ManagerBase.java

```diff
@@ -125,6 +132,9 @@ private void stopExecutor() {
     }
   }
 
+  protected void pause() {
```

Comment:
@Override?

Reply:
still missing @Override (?)

Reply:
No need to override here as it is abstract base class.